Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM node:8.12.0
```

Comment:
Could you take the latest version of node (which is called `carbon`)?

Reply:
Done!